sources/manticore-worker/Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM manticoresearch/helm-balancer:5.0.0.0 AS builder
-
 FROM manticoresearch/manticore:5.0.0
 
 RUN apt --fix-broken install -y && echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
-apt-get update && apt-get install -y curl php-cli php-mysqlnd php-curl wget && rm -rf /var/lib/apt/lists/*
-
-COPY --from=builder /var/www/localhost/ /etc/manticoresearch/
+apt-get update && apt-get update && apt -y install software-properties-common && add-apt-repository -y ppa:ondrej/php && apt-get update && \
+    apt-get install -y curl php7.4-cli php7.4-mysqlnd php7.4-curl wget git supervisor && rm -rf /var/lib/apt/lists/*
 
-RUN rm /etc/manticoresearch/observer.php
-
-COPY manticore.conf shutdown.sh starter.sh replica.php /etc/manticoresearch/
+COPY composer.json manticore.conf shutdown.sh starter.sh replica.php /etc/manticoresearch/
+COPY supervisord.conf /etc/supervisord.conf
 
 RUN wget https://repo.manticoresearch.com/repository/morphology/en.pak.tgz https://repo.manticoresearch.com/repository/morphology/de.pak.tgz https://repo.manticoresearch.com/repository/morphology/ru.pak.tgz -P /tmp && \
 tar -xvzf /tmp/en.pak.tgz -C /usr/share/manticore/ && tar -xvzf /tmp/ru.pak.tgz -C /usr/share/manticore/ && tar -xvzf /tmp/de.pak.tgz -C /usr/share/manticore/ && \
 chown -R manticore:manticore /var/log/manticore/ /var/lib/manticore/ /var/run/manticore /etc/manticoresearch/ /usr/share/manticore/ /dev/stdout
 
+RUN cd /etc/manticoresearch/ && \
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
+php -r "if (hash_file('sha384', 'composer-setup.php') === '55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
```

Comment:
Looks dangerous since as soon as they update the composer it will break since a new version will not match the old hash. There should be another way to use composer in Dockerfile

Reply:
This is the normal installation process described [here](https://getcomposer.org/download/ )